.github/workflows/review.yml

```diff
@@ -45,3 +45,17 @@ jobs:
         env:
           ENVIRONMENT_URL: https://${{ env.K8S_NAMESPACE }}.${{ env.BASE_DOMAIN }}
           DOCKER_IMAGE_NAME: ${{ env.REPO_NAME }}-review
+
+      - name: Setup Tunnistamo
+        uses: City-of-Helsinki/setup-tunnistamo-action@main
+        with:
+          kubeconfig_raw: ${{ env.KUBECONFIG_RAW}}
+          review_namespace: ${{ env.K8S_NAMESPACE }}
+          api_scope: https://api.hel.fi/auth/jassariapi
+
+      - name: Setup Tunnistamo
+        uses: City-of-Helsinki/setup-tunnistamo-action@main
+        with:
+          kubeconfig_raw: ${{ env.KUBECONFIG_RAW}}
+          review_namespace: ${{ env.K8S_NAMESPACE }}
+          api_scope: https://api.hel.fi/auth/helsinkiprofile
```

Comment:
I guess it's not really possible in `setup-tunnistamo-action`, but I think the frontend would need two scopes `https://api.hel.fi/auth/jassariapi` and `https://api.hel.fi/auth/helsinkiprofile`? :thinking:

Reply:
That's why the action is called twice, as one call is able to connect the client only to one scope. So the review client should be correctly having 2 scopes.

Reply:
Ah, indeed, all good here then. :+1: